new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorTwoDomains.java

```diff
@@ -0,0 +1,690 @@
+                            new V1VolumeMount()
+                                .name(pvName) // location to write domain
+                                .mountPath("/shared"))) // mounted under /shared inside pod
+                        .addCommandItem("/bin/sh") //call wlst.sh script with py and properties file
+                        .addArgsItem("/u01/oracle/oracle_common/common/bin/wlst.sh")
+                        .addArgsItem("/u01/weblogic/create-domain.py")
+                        .addArgsItem("-skipWLSModuleScanning")
+                        .addArgsItem("-loadProperties")
+                        .addArgsItem("/u01/weblogic/domain.properties")))
+                    .volumes(Arrays.asList(
+                        new V1Volume()
+                            .name(pvName)
+                            .persistentVolumeClaim(
+                                new V1PersistentVolumeClaimVolumeSource()
+                                    .claimName(pvcName)),
+                        new V1Volume()
+                            .name("create-weblogic-domain-job-cm-volume")
+                            .configMap(
+                                new V1ConfigMapVolumeSource()
+                                    .name(domainScriptConfigMapName))))  //ConfigMap containing domain scripts
+                    .imagePullSecrets(isUseSecret ? Arrays.asList(
+                        new V1LocalObjectReference()
+                            .name(OCR_SECRET_NAME))
+                        : null))));
+
+    logger.info("Running a job {0} to create a domain on PV for domain {1} in namespace {2}",
+        jobBody.getMetadata().getName(), domainUid, domainNamespace);
+    runCreateDomainJob(jobBody, domainNamespace);
+
+  }
+
+  /**
+   * Create a properties file for WebLogic domain configuration.
+   * @param wlstPropertiesFile path of the properties file
+   * @param domainUid the WebLogic domain for which the properties file is created
+   * @throws FileNotFoundException when properties file path not found
+   * @throws IOException when writing properties fails
+   */
+  private void createDomainProperties(Path wlstPropertiesFile,
+                                      String domainUid) throws FileNotFoundException, IOException {
+    // create a list of properties for the WebLogic domain configuration
+    Properties p = new Properties();
+
+    p.setProperty("domain_path", "/shared/domains");
+    p.setProperty("domain_name", domainUid);
+    p.setProperty("cluster_name", clusterName);
+    p.setProperty("admin_server_name", ADMIN_SERVER_NAME_BASE);
+    p.setProperty("managed_server_port", "8001");
+    p.setProperty("admin_server_port", "7001");
+    p.setProperty("admin_username", adminUser);
+    p.setProperty("admin_password", adminPassword);
+    p.setProperty("admin_t3_public_address", K8S_NODEPORT_HOST);
+    p.setProperty("admin_t3_channel_port", "" + getNextFreePort(32001, 32101));
```

Comment:
The same port needs to be used while creating the domain

Reply:
Just FYI... 
the port range has to be between 30,000 to 32,767

Reply:
I ran the test and the result showed that the admin_t3_channel_port here is a cluster IP. We can hard code the port number here.

Reply:
fixed